### PR TITLE
feat: Create and autofill address fields in inline-menu

### PR DESCRIPTION
**Pull request by Merkur39**

### CODE REVIEW

File: apps/browser/src/autofill/background/overlay.background.ts [hunk 1]
```diff
@@ -1030,13 +1035,33 @@ export class OverlayBackground implements OverlayBackgroundInterface {
...
+          ? {
+              fillOnlyTheseFieldQualifiers: [
+                "identityAddress1",
+                "identityCity",
+                "identityCountry",
+                "identityPostalCode",
+                "identityState",
+              ],
+            }
+          : {}),
```
Comment (zatteo) [inline apps/browser/src/autofill/background/overlay.background.ts:1064]: Don't know what to think about this code 😅

File: apps/browser/src/autofill/overlay/inline-menu/pages/list/autofill-inline-menu-list.ts [hunk 2]
```diff
@@ -338,18 +355,30 @@ export class AutofillInlineMenuList extends AutofillInlineMenuPageElement {
...
+        const key = Object.keys(input)[0] as CozyContactFieldNames;
+        inputValues[key] = input[key].value;
+      }
+
+      if (selectElement) {
+        inputValues = {
+          ...inputValues,
+          ...(selectElement && JSON.parse(selectElement.value)),
+        };
+      }
```
Comment (zatteo) [inline apps/browser/src/autofill/overlay/inline-menu/pages/list/autofill-inline-menu-list.ts:375]:
```
if (selectElement) {
  inputValues.selectElement = JSON.parse(selectElement.value);
}
```

would work also ?

**Review by zatteo — APPROVED** [review]

File: apps/browser/src/autofill/overlay/inline-menu/pages/list/autofill-inline-menu-list.ts (hunk 2 above)
Comment (Merkur39) [reply apps/browser/src/autofill/overlay/inline-menu/pages/list/autofill-inline-menu-list.ts:376]: The `label` and `type` properties are retrieved here, and I've added a comment

File: apps/browser/src/autofill/background/overlay.background.ts (hunk 1 above)
Comment (Merkur39) [reply apps/browser/src/autofill/background/overlay.background.ts]: Indeed, corrected

File: apps/browser/src/autofill/background/overlay.background.ts [hunk 3]
```diff
@@ -1030,13 +1035,33 @@ export class OverlayBackground implements OverlayBackgroundInterface {
...
         cipher,
-        fieldQualifier,
-        newAutofillValue,
+        inputValues,
         i18nService: this.i18nService,
+        logService: this.logService,
       });
+
+      const isAddress = Object.keys(inputValues).includes("street");
+      // If is an address, we need set the value of the field to the formatted address, because
```
[the remark this answers is not in the document]
Comment (Merkur39) [reply apps/browser/src/autofill/background/overlay.background.ts]: Oups, fix!

**Pull request merged**